docs/persistence.rst

```diff
@@ -0,0 +1,81 @@
+.. _persistence:
+
+Secure persistence with skops
+=============================
+
+.. warning::
+
+   This feature is very early in development, which means the API is
+   unstable and it is **not secure** at the moment. Therefore, use the same
+   caution as you would for ``pickle``: Don't load from sources that you
+   don't trust. In the future, more security will be added.
+
+Skops offers a way to save and load sklearn models without using :mod:`pickle`.
+The ``pickle`` module is not secure, but with skops, you can securely save and
+load sklearn models without using ``pickle``.
+
+Pickle is the standard serialization format for sklearn and for Python in
+general. One of its biggest advantages is it can be used for almost all Python
+code but this flexibility also makes it inherently insecure. This is because
+loading certain types of objects requires the ability to run arbitrary code,
+which can be misused for malicious purposes. For example, an attacker can use it
+to steal secrets from your machine or install a virus. As the Python docs say:
+
+    The pickle module is not secure. Only unpickle data you trust. It is
+    possible to construct malicious pickle data which will execute arbitrary
+    code during unpickling. Never unpickle data that could have come from an
+    untrusted source, or that could have been tampered with.
```

Comment:
maybe put this in a warning tag? (https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html#warnings)